core/src/main/java/org/apache/calcite/util/CompoundClosableAllocation.java

```diff
@@ -80,5 +67,3 @@ public boolean hasAllocations() {
     return !allocations.isEmpty();
   }
 }
-
-// End CompoundClosableAllocation.java
```

Comment:
Unfortunately, removal of this line will break checkstyle

Reply:
Hmm.  That's an odd requirement and not very Java-y.  Will update soon.

Reply:
Please feel free to comment on http://mail-archives.apache.org/mod_mbox/calcite-dev/201809.mbox/%3CCAB=Je-FYQ=R=LjZ4+=EviE72MM_5xqAJk1-Bv=XAv1EipVx4TA@mail.gmail.com%3E  and/or https://issues.apache.org/jira/browse/CALCITE-490